torch/quantization/observer.py

```diff
@@ -160,12 +160,28 @@ def _calculate_qparams(self, min_val, max_val):
 
     with_args = classmethod(_with_args)
 
+    def _save_to_state_dict(self, destination, prefix, keep_vars):
+        super(ObserverBase, self)._save_to_state_dict(destination, prefix, keep_vars)
+
+        destination[prefix + 'dtype'] = self.dtype
```

Comment:
same comment - state_dict is runtime state serialization, it shouldn't include arguments to constructor